Firestore/core/src/firebase/firestore/local/leveldb_transaction.h

```diff
@@ -144,6 +144,14 @@ class LevelDbTransaction {
    */
   static const leveldb::WriteOptions& DefaultWriteOptions();
 
+  leveldb::DB* database() const {
```

Comment:
I think ideally we wouldn't need this, and this points to passing around the wrong object.

Reply:
Removed.